Makefile

```diff
@@ -164,3 +164,23 @@ name-collision:
 
 composer-dependency-analyser:
 	php vendor/bin/composer-dependency-analyser --config build/composer-dependency-analyser.php
+
+.PHONY: infection
+infection:
+	git -C build-infection pull || git clone https://github.com/phpstan/build-infection.git
+	git -C build-infection fetch origin && git -C build-infection reset --hard origin/1.x
+	composer install --working-dir build-infection --no-interaction --no-progress
+	php build-infection/bin/infection-config.php --source-directory='build/PHPStan/Build'> infection.json5
```

Comment:
Why just `--source-directory='build/PHPStan/Build'`? That's not what I expect when I'd run `make infection`.